<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CutDialog</class>
 <widget class="QDialog" name="CutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>裁剪</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QRadioButton
{
	color:rgb(221, 221, 221);
}
</string>
  </property>
  <widget class="QLabel" name="topLeftLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>左：</string>
   </property>
  </widget>
  <widget class="QLabel" name="topRightLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>右：</string>
   </property>
  </widget>
  <widget class="QLabel" name="BottomLeftLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>上：</string>
   </property>
  </widget>
  <widget class="QLabel" name="BottomRightLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>下：</string>
   </property>
  </widget>
  <widget class="QSlider" name="topLeftSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>26</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="topRightSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>56</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="bottomLeftSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>86</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="bottomRightSlider">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>116</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="topLeftValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="topRightValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="BottomLeftValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="BottomRightValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conformButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
